iaso/api/serializers.py

```diff
@@ -143,6 +152,15 @@ def get_instances_count(self, org_unit):
         else:
             return org_unit.instance_set.filter(~Q(file="") & ~Q(device__test_device=True) & ~Q(deleted=True)).count()
 
+    def get_creator(self, org_unit):
```

Comment:
I'm not familiar with the code, but I'm not sure why we redefine here an improved(?) version of get_creator() here. Would the improved version work everywhere (in that case I suggest moving it to the parent class and letting others inherit it?). Or is there a good reason for a different behavior in `OrgUnitSearchSerializer`? If so, can we add a comment explaining the "why"? thanks!